plugins/redis32/src/main/resources/plugin.properties

```diff
@@ -0,0 +1,3 @@
+service.default org.radargun.service.RedisService
+service.server org.radargun.service.RedisService
+service.redis-client org.radargun.service.RedisClientService
```

Comment:
I based it on hazelcast, where we call the client hazelcast-client. Jedis looks better, I'll change it.